packages/office-ui-fabric-react/src/components/Calendar/Calendar.test.tsx

```diff
@@ -70,26 +69,22 @@ describe('Calendar', () => {
     let defaultDate = new Date('invalid');
 
     // Act
-    try {
-      let renderedComponent = ReactTestUtils.renderIntoDocument(
-        <Calendar
-          strings={ dayPickerStrings }
-          isMonthPickerVisible={ true }
-          value={ defaultDate }
-        />) as Calendar;
 
-      let today = ReactTestUtils.findRenderedDOMComponentWithClass(renderedComponent, 'ms-DatePicker-day--today') as HTMLElement;
+    let renderedComponent = mount(
+      <Calendar
+        strings={ dayPickerStrings }
+        isMonthPickerVisible={ true }
+        value={ defaultDate }
+      />);
 
-      expect(+today.innerText).to.be.equal(new Date().getDate());
-    } catch (err) {
-      expect.fail(err, null, 'Encountered error trying to render a Calendar with an invalid date');
-    }
+    let today = renderedComponent.find('.ms-DatePicker-day--today');
+    expect(+today.text()).toEqual(new Date().getDate());
```

Comment:
Removed `try/catch` because Calendar is supposed to handle invalid dates and default to today's date if passed an invalid date